src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -54,132 +54,11 @@ Copyright (c) .NET Foundation. All rights reserved.
-          FrameworkReferenceToUse="Microsoft.AspNetCore.App">
-      <Output TaskParameter="ShouldRemovePackageReference" PropertyName="_ShouldRemoveAspNetCoreAll" />
-    </CheckIfPackageReferenceShouldBeFrameworkReference>
-
-    <!-- NETSDK1079: The Microsoft.AspNetCore.All package is not supported when targeting .NET Core 3.0 or higher.
-         A FrameworkReference to Microsoft.AspNetCore.App should be used instead, and will be implicitly included by Microsoft.NET.Sdk.Web. -->
-    <NETSdkError Condition="'$(_ShouldRemoveAspNetCoreAll)' == 'true'"
-                 ResourceName="AspNetCoreAllNotSupported" />
-
-    <CheckIfPackageReferenceShouldBeFrameworkReference
-      PackageReferences="@(PackageReference)"
-      FrameworkReferences="@(FrameworkReference)"
-      PackageReferenceToReplace="Microsoft.AspNetCore.App"
-      FrameworkReferenceToUse="Microsoft.AspNetCore.App">
-      <Output TaskParameter="ShouldRemovePackageReference" PropertyName="_ShouldRemoveAspNetCoreApp" />
-      <Output TaskParameter="ShouldAddFrameworkReference" PropertyName="_ShouldAddAspNetCoreAppFrameworkReference" />
-    </CheckIfPackageReferenceShouldBeFrameworkReference>
-
-    <ItemGroup>
-      <PackageReference Remove="Microsoft.AspNetCore.App" Condition="'$(_ShouldRemoveAspNetCoreApp)' == 'true'" />
-      <FrameworkReference Include="Microsoft.AspNetCore.App" Condition="'$(_ShouldAddAspNetCoreAppFrameworkReference)' == 'true'" />
-    </ItemGroup>
-
-    <!-- NETSDK1080: A PackageReference to Microsoft.AspNetCore.App is not necessary when targeting .NET Core 3.0 or higher.
-         If Microsoft.NET.Sdk.Web is used, the shared framework will be referenced automatically. Otherwise, the PackageReference
-         should be replaced with a FrameworkReference. -->
-
-    <NETSdkWarning Condition="'$(_ShouldRemoveAspNetCoreApp)' == 'true'"
-                   ResourceName="AspNetCoreUsesFrameworkReference" />
-
-  </Target>
-
-  <Target Name="ApplyImplicitVersions"
-          BeforeTargets="_CheckForInvalidConfigurationAndPlatform;CollectPackageReferences;ProcessFrameworkReferences"
-          DependsOnTargets="UpdateAspNetToFrameworkReference;CheckForImplicitPackageReferenceOverrides"
-          Condition="'@(PackageReference)' != ''">
-
-    <ApplyImplicitVersions TargetFrameworkVersion="$(_TargetFrameworkVersionWithoutV)"
-                           TargetLatestRuntimePatch="$(TargetLatestRuntimePatch)"
-                           PackageReferences="@(PackageReference)"
-                           ImplicitPackageReferenceVersions="@(ImplicitPackageReferenceVersion)">
-      <Output TaskParameter="PackageReferencesToUpdate" ItemName="PackageReferenceToUpdate" />
-      <Output TaskParameter="SdkBuildWarnings" ItemName="SdkBuildWarning" />
-    </ApplyImplicitVersions>
+  <!-- Import the properties and item groups pulled from Microsoft.NET.Sdk.DefaultItems.targets to be shared by the WAP project system -->
```

Comment:
Rather than comment about this at the import site, I think it would make sense to explain this in the header section of the file that's being imported.